src/controllers/question-controller.js

```diff
@@ -0,0 +1,19 @@
+const Questions = require('../models/questions');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Martha-Miroslava/APIBiziidiDiidxaza/pullRequest?prid=8292860)
```suggestion
const Questions = require("../models/questions");
```